.github/workflows/ros2.yml

```diff
@@ -30,6 +30,36 @@ jobs:
         ros_distro: [ humble ]
 
     steps:   
+    - name: Install gsl-lite
+      run: |
+        git clone https://github.com/gsl-lite/gsl-lite && cd gsl-lite
+        mkdir build && cd build
+        cmake .. && make -j8
+        sudo make install
+
+    - name: Install Catch2
+      run: |
+        git clone https://github.com/catchorg/Catch2 && cd Catch2
+        mkdir build && cd build
+        cmake .. && make -j8
+        sudo make install
+
+    - name: Install fmt
+      run: |
+        git clone https://github.com/fmtlib/fmt && cd fmt
+        mkdir build && cd build
+        cmake -DFMT_TEST=OFF ..
+        make -j8
+        sudo make install
+
+    - name: Install mp-units
+      run: |
+        git clone https://github.com/mpusz/mp-units && cd mp-units
+        mkdir build && cd build
+        cmake -DMP_UNITS_AS_SYSTEM_HEADERS=ON -DMP_UNITS_BUILD_LA=OFF ..
+        make -j8
+        sudo make install
+
```

Comment:
You could use a Conan package manager for those.

See: https://mpusz.github.io/mp-units/latest/getting_started/installation_and_usage/#conan-cmake-live-at-head

Reply:
Thank you, I may update this. To be honest, I've never worked with `conan` before and it feels a bit uncomfortable. On the other side, this process that's written down here I understand down to the last bit ;). Is there any way to disable the build of the examples?

Reply:
You should use CMakeLists.txt from the `./src` subdirectory as mentioned here: https://mpusz.github.io/mp-units/2.1/getting_started/installation_and_usage/#copy-cmake and here: https://mpusz.github.io/mp-units/2.1/getting_started/installation_and_usage/#repository-structure-and-dependencies